### PR TITLE
Explicit warning not to try anything other running on Linux and on the same instance

### PR DESCRIPTION
**Pull request by Krysztophe**
From our discussion these last days, a big note in the installation note.

### CODE REVIEW

File: docs/installation.rst [hunk 1]
```diff
@@ -9,14 +9,18 @@ you may want to use trusted certificate and other enhancement.
...
-- PostgreSQL 9.4+, listening on UNIX socket, on the same host. Check with ``sudo -u postgres psql``.
+- PostgreSQL 9.4+, running on Linux and listening on UNIX socket. Check with ``sudo -u postgres psql``.
 - openssl.
 - Python 2.6+ or 3.5+. Check with ``python --version``.
 - A running temBoard UI.
 - bash and sudo for setup script.
 
+.. note::
+
+  Any configuration other than the agent on the same Linux server than the PostgreSQL instance is not supported. The agent will refuse to run.
```
Comment (bersace) [inline docs/installation.rst]: Reformulation : "temBoard agent must run on same host as PostgreSQL instance. Running agent on remote host is not supported yet."

File: docs/installation.rst [hunk 2]
```diff
@@ -9,14 +9,18 @@ you may want to use trusted certificate and other enhancement.
 Prerequisites
 =============
 
-In order to run temBoard agent, you require:
+In order to run temBoard agent, you need:
 
-- PostgreSQL 9.4+, listening on UNIX socket, on the same host. Check with ``sudo -u postgres psql``.
+- PostgreSQL 9.4+, running on Linux and listening on UNIX socket. Check with ``sudo -u postgres psql``.
```
Comment (bersace) [inline docs/installation.rst]: Worth a dedicated entry for OS.

File: docs/installation.rst (hunk 1 above)
Comment (Krysztophe) [reply docs/installation.rst]: Done in 2bd89a41c6daec31496b604bba5e2007485dadd5

File: docs/installation.rst (hunk 2 above)
Comment (Krysztophe) [reply docs/installation.rst]: done in 2bd89a41c6daec31496b604bba5e2007485dadd5

**Pull request merged**